src/adapters/komoona.js

```diff
@@ -0,0 +1,75 @@
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+var adloader = require('../adloader');
+
+//Version: 1.2
+
+var KomoonaAdapter = function KomoonaAdapter() {
+  var DATA_VER = '0.1';
+  var KOMOONA_URL = "//s.komoona.com/kb/{DATA_VER}/kmn_sa_kb_c.{hbid}.js";
+
+  var KOMOONA_BIDDER_NAME = 'komoona';
+  var _bidsMap = {};
+
+  function _callBids(params) {
+    var kbConf = {
+      ts_as: new Date().getTime(),
+      hb_placements: [],
+      kb_callback: _bid_arrived
+    };
+
+    var bids = params.bids || [];
+    if (!bids || !bids.length) {
+      return;
+    }
+
+    var dataVersion = DATA_VER;
+    
+    for(var i in bids) {
+      var currentBid = bids[i];
+      _bidsMap[currentBid.params.placementId] = currentBid;
+      kbConf.hdbdid = kbConf.hdbdid || currentBid.params.hbid;
+      kbConf.encode_bid = kbConf.encode_bid || currentBid.params.encode_bid;
+      kbConf.hb_placements.push(currentBid.params.placementId);
+      if (currentBid.params.dataVersion)
+        dataVersion = currentBid.params.dataVersion;
+    }
+
+    var scriptUrl = KOMOONA_URL.replace('{DATA_VER}', dataVersion).replace('{hbid}', kbConf.hdbdid);
+
+    adloader.loadScript(scriptUrl, function() {
```

Comment:
can the URL be cached? If so, please add `true` to the 3rd param of `loadScript(url, callback, true);` This will enable caching of the URL.